### DISCUSSION TITLE
PR suggestion gets incorrectly formatted as MathJax

### DISCUSSION THREAD

**Discussion by biinari** [community/community#31479] [Pull Requests]
In a pull request, when I have made a suggestion for a change which happens to include dollar signs for variables, the comment formats it as MathJax.

![pr_suggestion_2022-08-31_11-57-03](https://user-images.githubusercontent.com/148009/187663476-b620dbe9-ca18-4c6c-8ec1-eb821b98b106.png)

There is a [suggestion for working around this problem in comments](https://github.com/orgs/community/discussions/17522#discussioncomment-3066645). This seems to be impossible to use as a workaround in a PR suggestion block though.

My suggested solution would be to ensure that when suggestions are rendered in a comment, they are always treated as source code with no attempt to look for MathJax (or any other markdown treatments).

**Comment by dpvc** [comment]
The GitHub team should be able to add the appropriate class to the `tex2jax_ignore` value in their MathJax configuration to make that happen easily enough.

**Reply by biinari** [reply]
That will be perfect, thanks!

**Reply by dpvc** [reply]
It will be up to them to do it, but there is a configuration option to make it easy.  They just have to make the change.
